src/engine/lumix.h

```diff
@@ -29,7 +36,7 @@ typedef unsigned int uint32;
 typedef long long int64;
```

Comment:
I know, I used to do that, but because of it cstdint would be included in every file, which can slow down compilation. Also I do not like "_t" postfix. Finally, one day I plan to get rid of all possible [external dependencies](http://www.catch22.net/tuts/reducing-executable-size)